doc/nrf/doc_styleguide.rst

```diff
@@ -81,7 +81,9 @@ domains:
 
 .. note::
    The ``:cpp:enum:`` and ``:cpp:enumerator:`` domains do not generate a link
-   due to an issue with Sphinx and breathe.
+   due to an `issue <https://github.com/michaeljones/breathe/issues/437>`_ with
```

Comment:
All links should go into the links.txt file - that makes it easier to keep them up-to-date.